myning/chapters/mine/actions.py

```diff
@@ -119,6 +120,7 @@ def content(self):
         return content_table
 
     def fight(self):
+        stats = Stats()
```

Comment:
by importing the TUI after initializing objects, we can now place these references at the top of the file instead of in every function